Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/v1/types.go

```diff
@@ -2597,7 +2597,19 @@ type SecurityContextConstraints struct {
 
 	// AllowPrivilegedContainer determines if a container can request to be run as privileged.
 	AllowPrivilegedContainer bool `json:"allowPrivilegedContainer" description:"allow containers to run as privileged"`
+	// Initial capabilities are defaulted into containers.
+	// Initial capabilities will be enforced with the following rules
+	// 1.  InitialCapabilities.Drop and AllowedCapabilities cannot conflict (you may not require
+	//		a cap to be dropped but also let it be requested)
+	// 2.  Capabilities in InitialCapabilities.Add will be added to the container unless the container specifically
+	// 		requests it be dropped.
+	// 3.  Capabilities in InitialCapabilities.Drop will be dropped from the container
+	// 4.  Containers that request to add a capability that is required to be dropped will fail
+	//		validation.
+	InitialCapabilities Capabilities `json:"initialCapabilities" description:"capabilities that will be added or dropped"`
```

Comment:
I know it is bike shedding, but 'assigned' sounds to me like it would blindly overwrite things. Which it only sort of does. You don't end up with the 'assigned' caps. You end up with some amalgamation.

I'm not arguing for Initial either, I'll try to think of something to propose instead of just being a negative nancy...